packages/utils/src/phoneNumbers.ts

```diff
@@ -152,6 +164,10 @@ export function parsePhoneNumber(
       ? {
           e164Number: phoneUtil.format(parsedNumber, PhoneNumberFormat.E164),
           displayNumber: handleSpecialCasesForDisplay(parsedNumber, parsedCountryCode),
+          displayNumberInternational: phoneUtil.format(
+            parsedNumber,
+            PhoneNumberFormat.INTERNATIONAL
+          ),
```

Comment:
We might want international formatting throughout the app to prevent confusion.
See also the existing comment: https://github.com/celo-org/celo-monorepo/blob/35007d248bd8e0ad8e84107250985ad4b8d5088c/packages/utils/src/phoneNumbers.ts#L186-L188
I've added it as a new property to prevent changing the existing behavior.